common/templates/context_funcs.go

```diff
@@ -402,6 +402,32 @@ func (c *Context) tmplEditMessage(filterSpecialMentions bool) func(channel inter
 	}
 }
 
+func (c *Context) tmplHandlePinMessage(mode bool) func(channel, msgID interface{}) (string, error) {
```

Comment:
```suggestion
func (c *Context) tmplPinMessage(mode bool) func(channel, msgID interface{}) (string, error) {
```
For consistency with similar higher-order functions.